modules/osdk-golang-build-run.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-build-run_{context}"]
+= Build and run the Operator
+
+After implementing the new controller, you can build and run your Operator.
+
+.Procedure
+
+. Register the CRD with the Kubernetes API server:
++
+[source,terminal]
+----
+$ make install
+----
+
+. By default, Go-based Operator projects are scaffolded with unit tests that utilize the `envtest` framework from the `controller-runtime` libraries to simulate the API portions of a real cluster. These tests require certain Kubernetes server binaries be present locally, including `kubectl`, `kube-apiserver`, and etcd.
```

Comment:
It seems like both of these substeps already happen by default now (the script seemed to run automatically during an earlier step, and the Makefile looked like it already had the test target updated), however it was mentioned in the upstream docs. Is this step still needed for v1.4.0?